grpc/oob/machine/machine.go

```diff
@@ -225,24 +264,29 @@ func (m Action) PowerSet(ctx context.Context, action string) (result string, err
 	log.Info("connected to BMC", logMetadata(client.GetMetadata())...)
 	m.SendStatusMessage("connected to BMC")
 
+	// fetch the current power state that will be returned by "status"
+	// or used for control in cycle, and is always sent in traces
+	currentPowerState, err := client.GetPowerState(ctx)
```

Comment:
After the call to GetPowerState (and also to SetPowerState elsewhere) the client metadata would be valuable to annotate the span